script.js

```diff
@@ -22,6 +22,20 @@ $(document).ready(function() {
 });
 
 function handleResponse(response) {
-  console.log(response);
+  for (var i=0; i<response.data.length; i++) {
+    var url= response.data[i].images.standard_resolution.url;
+    var caption=response.data[i].caption.text;
+   // console.log(url);
+    $("#list").append("<img src=" + url + "/>");
+    $("#list").append("<div>" + caption + "</div>");
```

Comment:
For stage 4 adding an id to caption will make it easier to display the positivity ranking